### PR TITLE
[DOCFIX]Update Configuring-Alluxio-with-OSS.md of the Chinese version

### PR DESCRIPTION
**Pull request by granttoyu**

### CODE REVIEW

File: docs/cn/Configuring-Alluxio-with-OSS.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ priority: 4
 
 {% include Configuring-Alluxio-with-OSS/oss-access.md %}
 
-其中, `fs.oss.accessKeyId `和`fs.oss.accessKeySecret`分别对应于阿里云[AccessKeys管理界面](https://ak-console.aliyun.com/#/accesskey)中实际的`Access Key ID`和`Access Key Secret`，`fs.oss.endpoint`则是Bucket概述中所说的Bucket所在的范围，其可能的取值有`oss-us-west-1.aliyuncs.com `，`oss-cn-shanghai.aliyuncs.com`等。
+此处, `fs.oss.accessKeyId `和`fs.oss.accessKeySecret`分别为`Access Key ID`字符串和`Access Key Secret`字符串，均受阿里云[AccessKeys管理界面](https://ak-console.aliyun.com/#/accesskey)管理；`fs.oss.endpoint`是Bucket概述中所说的Bucket的末端点，其可能的取值比如`oss-us-west-1.aliyuncs.com `，`oss-cn-shanghai.aliyuncs.com`。
```
Comment (saltylin) [inline docs/cn/Configuring-Alluxio-with-OSS.md]: "末端点" is not very suitable here. We can simply leave "endpoint" untranslated.

**Comment by saltylin** [comment]
@granttoyu Thanks very much for your translation. I have left a minor comment. The other LGTM

**Comment by aaudiber** [comment]
Jenkins, ok to test

**Comment by apc999** [comment]
LGTM thanks!

**Pull request merged**